src/ToDoList.js

```diff
@@ -18,107 +16,97 @@ import {
   IconButton,
 } from "@mui/material";
 
-const containerStyle = {
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-  width: "500px",
-  height: "600px",
-  backgroundColor: "lightGrey",
-  borderRadius: "30px",
-  margin: "0 auto",
-  marginTop: "30px",
-};
-
-const titleStyle = {
-  color: "black",
-  variant: "h1",
-  width: "300px",
-  height: "50px",
-  //borderRadius: "20px",
-  //backgroundColor: "lightBlue",
-  margin: "0 auto",
-  fontWeight: "bold",
-  //justifyContent: "center",
-  //alignItems: "center",
-};
-
-const inputContainerStyle = {
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-  //height: "40px",
-};
-
-/*const taskItemStyle = {
-  width: "200px",
-  display: "flex",
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-};*/
-
-const completedTaskStyle = {
-  textDecoration: "line-through",
-  textDecorationStyle: "solid",
-  textDecorationThickness: "2px",
-  color: "black",
-};
-
-/*const deleteButtonStyle = {
-  fontSize: "10px",
-  padding: "3px 3px",
-};*/
+const containerStyle = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  flex-direction: column;
+`;
+
+const titleStyle = css`
+  color: black;
+  font-size: 24px;
+  width: 300px;
+  height: 50px;
+  border-radius: 20px;
+  background-color: lightBlue;
+  margin: 0 auto;
+  font-weight: bold;
+  margin-top: 30px;
+  display: flex;
+  flex-direction: column;
+`;
+
+const inputContainerStyle = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 30px;
+`;
+
+const completedTaskStyle = css`
+  text-decoration: line-through;
+  text-decoration-style: solid;
+  text-decoration-thickness: 2px;
+  color: black;
+`;
 
 const ToDoList = () => {
+  const [tasks, setTasks] = useState([]);
+  const [taskInfo, setTaskInfo] = useState("");
+  const [filter, setFilter] = useState("all");
+
   useEffect(() => {
     const storedTasks = localStorage.getItem("tasks");
     if (storedTasks) {
       setTasks(JSON.parse(storedTasks));
     }
   }, []);
 
-  const [tasks, setTasks] = useState(() => {
-    const storedTasks = localStorage.getItem("tasks");
-    return storedTasks ? JSON.parse(storedTasks) : [];
-  });
-
-  const [taskInfo, setTaskInfo] = useState("");
-  const [filter, setFilter] = useState("all");
-
-  useEffect(() => {
-    localStorage.setItem("tasks", JSON.stringify(tasks));
-  }, [tasks]);
-
   const handleInputChange = (event) => {
     setTaskInfo(event.target.value);
   };
+
   const addTask = () => {
     if (taskInfo.trim() !== "") {
+      const isTaskExists = tasks.some((task) => task.task === taskInfo.trim());
```

Comment:
Are you sure the `.trim()` is needed here ? you're already checking that on the `addTask` function,
Correct me if I'm wrong :)

Reply:
I use .trim() to remove leading and trailing whitespace from a string and compare the input of the task. 
For example if there is an existing task: "a" and I try to add a new task: "    a   " if I will not use .trim() it will not recognize that the task already exists. Is there a better way to achieve this?